tests/test_physics.py

```diff
@@ -593,7 +593,17 @@ def getRestPositions(articulated_object):
 
 def getRandomPositions(articulated_object):
     r"""Constructs a random pose vector for an ArticulatedObject with unit quaternions for spherical joints."""
-    rand_pose = np.random.uniform(-1, 1, len(articulated_object.joint_positions))
+    lower_limits = articulated_object.get_joint_position_limits(upper_limits=False)
+    upper_limits = articulated_object.get_joint_position_limits(upper_limits=True)
```

Comment:
This seems like a counter_intuitive return for me. You could return a tuple that includes both, right?

Reply:
Yeah, that makes sense. I was trying to avoid extra work if user doesn't want both, but simplicity is probably best.